pkg/conduit/runtime.go

```diff
@@ -159,11 +159,9 @@ func newLogger(level string, format string) log.CtxLogger {
 	l, _ := zerolog.ParseLevel(level)
 	f, _ := log.ParseFormat(format)
 	logger := log.InitLogger(l, f)
-	logger = logger.CtxHook(
-		ctxutil.MessageIDLogCtxHook{},
-		ctxutil.RequestIDLogCtxHook{},
-		ctxutil.FilepathLogCtxHook{},
-	)
+	logger.Logger = logger.Hook(ctxutil.MessageIDLogCtxHook{})
+	logger.Logger = logger.Hook(ctxutil.RequestIDLogCtxHook{})
+	logger.Logger = logger.Hook(ctxutil.FilepathLogCtxHook{})
```

Comment:
It might look prettier if we chain the calls: `log.InitLogger().Hook().Hook().Hook()`.